release-common-lib/src/main/java/org/reactome/util/ensembl/EnsemblServiceResponseProcessor.java

```diff
@@ -222,16 +283,28 @@ public EnsemblServiceResult processResponse(HttpResponse response, URI originalU
 		}
 		else
 		{
-			// actually, this is not so strange - I think that http://rest.ensemblgenomes.org/ *never* returns X-RateLimit-Remaining; I think that header *only* comes from rest.ensembl.org
-			// So only log a message if we didn't get a rate limit from rest.ensembl.org - if it didn't come from rest.ensemblGENOMES.org, that's OK.
+			// actually, this is not so strange - I think that http://rest.ensemblgenomes.org/ *never* returns
+			// X-RateLimit-Remaining; I think that header *only* comes from rest.ensembl.org
```

Comment:
This statement will need to be tested from time to time cause Ensembl seems to be working pretty consistently on their rest APIs based on the release notes I've seen

Reply:
It actually may not be an issue anymore.  Since the last code update, rest.ensemblgenomes.org has been retired and merged into rest.ensembl.org.  I am working on a second pull request to do a bit of refactoring.